lib/server.js

```diff
@@ -285,9 +285,8 @@ export default class Server {
     }
 
     if (this.isTest()) {
-      if (config.testConfig) {
-        this.loadConfig(config.testConfig);
-      }
+      const testConfig = config.testConfig ?? function () {};
+      this.loadConfig(testConfig);
```

Comment:
I think I'm missing something, why do we need to call `loadConfig()` with a no-op if testConfig is undefined?  I get that it's one of the issues you identified originally, but it seems weird to call it with a no-op vs the config just being an optional parameter and a falsey value being handled in the `loadConfig` method itself.

Reply:
>the config just being an optional parameter and a falsey value being handled in the loadConfig method itself.

I'm fine with this change too. I'll update :+1:

Reply:
https://github.com/miragejs/miragejs/pull/1080/commits/ce991fd93f8c28c75ac234d92a46ee0c02a3f849